### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Please review my PR for  localStorage feature and custom checkbox

Screenshot 2022-04-13 123029

### CODE REVIEW

File: src/components/task-details-page/TaskDetailsPage.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function TaskDetailPage() {
         dispatchTaskList({ type: "TASK_COMPLETE",payload:{id} });
     };
 
-
+    useDocumentTitle(title)
```
Comment (sudipkundu999) [inline src/components/task-details-page/TaskDetailsPage.jsx:21]: I think you forgot to pass the actual title here

**Review by sudipkundu999 — APPROVED** [review]
The code looks good.
Good work. Good to merge

File: src/components/task-details-page/TaskDetailsPage.jsx (hunk 1 above)
Comment (bhushannaikaroskar) [reply src/components/task-details-page/TaskDetailsPage.jsx:21]: No actually I wanted to display the title based on the task title I want to change the title

**Review by srishtigautam21 — APPROVED** [review]
The looks clean and understandable
good to merge

**Pull request merged**